### DISCUSSION TITLE
Failed to send address update

### DISCUSSION THREAD

**Discussion by relu** [linkerd/linkerd2#10364] [Help] [answered]
Hello! I've started to see a lot of such errors coming from linkerd-destination lately (2.12.1 and 2.12.4):
```
Failed to send address update: rpc error: code = Canceled desc = context canceled
```

It happens very frequently and usually for the same service. Would appreciate any help or hints as to what might be causing this.

Example log message:
```
{"addr":":8086","component":"endpoint-translator","level":"error","msg":"Failed to send address update: rpc error: code = Canceled desc = context canceled","remote":{"IP":"<IP>","Port":56108,"Zone":""},"service":"<name>.<namespace>.svc.cluster.local:<port>","time":"2023-02-20T18:26:04Z"}

**Comment by jeremychase** [comment, marked as answer]
We are planning to [downgrade](https://github.com/linkerd/linkerd2/issues/10380) these log messages because we don't believe they are actionable. Are you seeing a particular problem as a result or are you mainly concerned with the message itself?

**Reply by relu** [reply]
Thanks for your reply, @jeremychase! I have not seen any actual issues, it is just an annoyance since it pollutes the logs, and I found it a bit strange that it's almost always the same service that this is reported for, which was a pattern that made me curious as to why this might happen.
